src/mgr.sol

```diff
@@ -281,6 +281,7 @@ contract TinlakeManager {
 
         // Return possible remainder to the owner
         dai.transfer(owner, dai.balanceOf(address(this)));
+        tab = sub(tab, mul(payBack, RAY));
```

Comment:
you're setting `tab` twice in this function now. What if you move these lines to after the wipe instead:
```
        (, uint256 art) = vat.urns(ilk, address(urn));
        (, uint256 rate, , ,) = vat.ilks(ilk);
       tab = mul(art, rate);
``

Reply:
hmm, although you do need it for the `payBack` calculation I suppose. A local variable would be better though, rather than two SSTOREs

Reply:
Yes exactly.  You are right, a local variable is cheaper 👍